### PR TITLE
ES-Country_fiches-2024

### PR DESCRIPTION
**Pull request by abadppa0**
Dear @fabiovinci 
As you have not yet closed the Country Fiche of Spain, he has taken the opportunity to include some more data.
This file contains the report of Spain on the implementation of the INSPIRE Directive regulations  in 2023. The last updated file is the one that contains all the information.
For any questions write to Paloma Abad (paloma.abad@cnig.es). Thank you.

### CODE REVIEW

**Comment by abadppa0** [comment]
Dear @fabiovinci,
As you have not yet closed the Country Fiche of Spain, he has taken the opportunity to include some more data.

Thank you,
Paloma

**Comment by fabiovinci** [comment]
Dear @abadppa0,

I fixed some typos and the PR is ready to merge.
Please confirm if the file [ES/Country_fiches/prueba](https://github.com/INSPIRE-MIF/INSPIRE-in-your-Country/pull/9/files/2dc904b8a2d95d7bbb66baef6a2dd2495040bf11#diff-796106e5569775ba913a50aeb187dadd84846056c79486c7025b899b914381b2) is necessary, if not I will remove it and merge the PR.
Thanks

**Comment by abadppa0** [comment]
Hello @fabiovinci 
- I have tried to delete "prueba" in my github but I don't know how to do it. Please delete "prueba".
- I´ve seen an "error" and I´ve taken the opportunity to change it.
- I just made you the final folk.

Grazie mille per tutto
Paloma

**Review by fabiovinci — APPROVED** [review]

File: ES/Country_fiches/prueba
[the remark this answers is not in the document]
Comment (fabiovinci) [reply ES/Country_fiches/prueba:1]: File was removed

**Comment by fabiovinci** [comment]
Hi @abadppa0,

thank you for the submission.

I deleted the empty file and I will merge the PR.

**Pull request merged**